release_tooling/src/release.py

```diff
@@ -265,6 +267,26 @@ def show_ecs(ctx, cluster_name):
     click.echo("\n".join(pprint_path_keyval_dict(paths)))
 
 
+@main.command()
+@click.argument('commit_ref', required=True)
+@click.pass_context
+def show_github(ctx, commit_ref):
+    github_repository = ctx.obj['github_repository']
+    github_metadata = GitHubMetadata(github_repository)
+
+    summaries = github_metadata.find_pull_requests(
+        commit_ref
+    )
+
+    click.echo('')
+    if not summaries:
+        click.echo("No matching pull requests found.")
+    else:
+        click.echo('Found pull requests:')
+        for summary in summaries:
+            click.echo("{number}: '{title}' {closed_at} {url}".format(**summary))
```

Comment:
nit: because the title can vary in length, maybe split this across multiple lines for readability? Something like:

```python
click.echo("#{number.rjust(4)}: {title!r}".format(**summary))
click.echo("      Closed at {closed_at}".format(**summary))
click.echo("      {url}".format(**summary))
```